test/test_readme_sample.py

```diff
@@ -72,7 +72,7 @@ def print(*args):
         vqc.fit(training_features, training_labels)
 
         score = vqc.score(test_features, test_labels)
-        print("Testing accuracy: {:0.2f}".format(score))
+        print(f"Testing accuracy: {score:0.2f}")
```

Comment:
As this code is part of the README file can we modify that the same way so this code that is tested is an exact copy still of the code we show there.

Reply:
My bad - I scrolled through the files and missed that it was there , duh! I see now its the second one in this list here. Thx!